src/erf.jl

```diff
@@ -362,6 +362,46 @@ function _erfinv(x::Float32)
     end
 end
 
+function _erfinv(x::Float16)
+    a = abs(x)
+    if a >= Float16(1)
+        if x == Float16(1)
+            return Inf16
+        elseif x == -Float16(1)
+            return -Inf16
+        end
```

Comment:
```suggestion
if a == Float16(1)
    return Inf16*a
end
```

Reply:
That's basically copied from the implementations of `Float32` and `Float64`. Regardless, I changed it in https://github.com/JuliaMath/SpecialFunctions.jl/pull/372/commits/04069add09d4cdb1ca4a8f176f40f44950cce685.